telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java

```diff
@@ -0,0 +1,17 @@
+package com.jakewharton.telecine;
+
+import android.content.Intent;
+import android.service.quicksettings.Tile;
+import android.service.quicksettings.TileService;
+
+public class TelecineTileService extends TileService {
+  @Override public void onClick() {
+    startActivity(new Intent(this, TelecineShortcutLaunchActivity.class));
+  }
+
+  @Override public void onStartListening() {
+    Tile tile = getQsTile();
+    tile.setState(Tile.STATE_ACTIVE);
```

Comment:
So right now there's no state syncing. It just shows the icon as solid white (active) at all times.